time t
duration deltat
int32 task
float64 x_a
float64 x_d1
float64 x_d2
float64 xd_a
float64 xd_d1
float64 xd_d2
float64 x_t
float64 Fenv
int32 direction
